tests/main_test.py

```diff
@@ -33,61 +50,50 @@ def teardown_class(cls):
         """
         pass
 
-    def test_get(self):
-        """
-        test
-        :return:
-        """
-        response = requests.get(self.base_url + '/test_get')
+    def ensure_status_code_msg(self, response, correct_status_code, correct_msg):
         data = response.json()
-        cond = (data['msg'] == "Smart Ledger API Endpoint: OK")
-        if not cond:
-            print('API is currently down')
-            sys.exit()
-        assert cond
+        cond_1 = response.status_code == correct_status_code
+        cond_2 = data['msg'] == correct_msg
+        assert cond_1 and cond_2, f"Expected status code {correct_status_code} with msg `{correct_msg}`, " \
+                                  f"got {response.status_code} with msg `{data['msg']}`"
 
-    def test_register(self):
+    def test_get(self):
         """
         test
         :return:
         """
-        content, status_code = self.do_post('/register', {})
-        assert status_code == 200
-        assert content['data']['sub'] == self.user['data']['sub']
+        response = requests.get(f'{self.base_url}/test_get')
+        self.ensure_status_code_msg(response, 200, "Smart Ledger API Endpoint: OK")
 
     def test_delete_and_register(self):
         """
         test
         :return:
         """
-
         # delete
-        content, status_code = self.do_post('/user/delete', {'sub': self.user['data']['sub']})
-        assert status_code == 200
-        assert content['msg'] == "User profile successfully deleted."
+        response = self.do_post('/user/delete', {'sub': self.user['data']['sub']})
+        self.ensure_status_code_msg(response, 200, "Successfully deleted the user profile.")
```

Comment:
@tdbucks after this check you may also want to have test to /user/info reporting that the user doesn't exist